_sub/network/external-dns/values/helm-install.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+  - ${gitops_apps_repo_url}/apps/${deploy_name}?ref=${gitops_apps_repo_branch}
+patchesStrategicMerge:
```

Comment:
Isn't this depracated and one should use patches instead.

`# Warning: 'patchesStrategicMerge' is deprecated. Please use 'patches' instead. Run 'kustomize edit fix' to update your Kustomization automatically.`